src/common/tiglcommonfunctions.cpp

```diff
@@ -1089,6 +1089,18 @@ gp_Pnt GetCenterOfMass(const TopoDS_Shape &shape)
      return centerPoint;
 }
 
+double GetArea(const TopoDS_Shape &shape)
+{
```

Comment:
Why did you commit this funtion again? You are mixing here two different feature / branches!